src/components/SearchForm/SearchForm.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import styles from "./SearchForm.module.scss";
+
+const SearchForm: React.FC<{
+  InitialSearchQuery: string;
```

Comment:
This is just a string prop, there is no need to capitalize it. In React it is common practice to capitalize components, types, interfaces, but not props of primitive types.